specs/agents/tracing-spans-drop-fast-exit.md

```diff
@@ -0,0 +1,81 @@
+[Agent spec home](README.md) > [Handling huge traces](tracing-spans-handling-huge-traces.md) > [Dropping fast exit spans](tracing-spans-drop-fast-exit.md)
+
+# Dropping fast exit spans
+
+If an exit span was really fast, chances are that it's not relevant for analyzing latency issues.
+Therefore, agents SHOULD implement the strategy laid out in this section to let users choose the level of detail/cost tradeoff that makes sense for them.
+If an agent implements this strategy, it MUST also implement [Collecting statistics about dropped spans](tracing-spans-dropped-stats.md).
+
+## `exit_span_min_duration` configuration
+
+Sets the minimum duration of exit spans.
+Exit spans that execute faster than this threshold are attempted to be discarded.
+
+The attempt fails if they lead up to a span that can't be discarded.
+Spans that propagate the trace context to downstream services,
+such as outgoing HTTP requests,
+can't be discarded.
+However, external calls that don't propagate context,
+such as calls to a database, can be discarded using this threshold.
+
+Additionally, spans that lead to an error can't be discarded.
+
+|                |            |
+|----------------|------------|
+| Type           | `duration` |
+| Default        | `1ms`      |
+| Central config | `true`     |
+
+TODO: should we introduce µs granularity for this config option?
+Adding `us` to all `duration`-typed options would create compatibility issues.
+So we probably want to support `us` for this option only.
```

Comment:
Why would adding a new duration suffix create compat issues? I'm missing something.

Reply:
IIUC a potential backward compatibility issue comes up if we add support for `us` suffix to existing options and `us` suffix is used in central configuration while there is a mix of old (not support `us` suffix) and new agents.